packages/web-components/src/components/table-of-contents/__stories__/table-of-contents.stories.react.tsx

```diff
@@ -24,6 +32,33 @@ export const Default = () => {
   );
 };
 
+export const WithHeadingContent = ({ parameters }) => {
+  const { menuRule } = parameters?.props?.Other ?? {};
+  return (
+    <>
+      <DDSTableOfContents>
+        <DDSImage slot="heading" alt="Alt text" default-src={imgLg1x1}>
+          <DDSImageItem media="(min-width: 1056px)" srcset={imgXlg16x9}></DDSImageItem>
+          <DDSImageItem media="(min-width: 672px)" srcset={imgLg16x9}></DDSImageItem>
+          <DDSImageItem media="(min-width: 400px)" srcset={imgMd16x9}></DDSImageItem>
+        </DDSImage>
+        {!menuRule ? null : <DDSHorizontalRule slot="menu-rule"></DDSHorizontalRule>}
+        {content()}
+      </DDSTableOfContents>
+    </>
+  );
+};
+
+WithHeadingContent.story = {
```

Comment:
One small thing that does need to change, is the title for this story appears to be title cased. Our storybook stories need to be sentence cased. "With heading content"